src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-sizing: border-box;
+  width: 100%;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin: 23px 10px;
+}
+
+.nav__item:first-child {
+  margin: 23px 10px 23px 0;
+}
+
+.nav__item:last-child {
+  margin: 23px 0 23px 10px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  color: black;
+  padding: 23px 0;
+  cursor: pointer;
```

Comment:
afaik, this is the default behaviour for links, no need to specify it explicitly